### DISCUSSION TITLE
Touchpad Scroll and Gestures Direction

### DISCUSSION THREAD

**Discussion by arshali2774** [kRHYME7/Hyde-cli#96] [Q&A]
Awsome thing you have built here. Coming from windows I noticed that for scrolling up and down and gestures for back and forward on browser has been inversed. On windows to scroll down we swipe up and vice-versa and to go back we swipe right. 
Is there way to change that so that my muscle memory can be unchanged. LOL.
Scroll wheel on mouse works fine.

**Comment by kRHYME7** [comment]
https://github.com/prasanthrangan/hyprdots/discussions/1965#discussioncomment-11132203
